flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/OracleE2eITCase.java

```diff
@@ -159,7 +159,7 @@ public void testOracleCDC() throws Exception {
                         "103,12-pack drill bits,12-pack of drill bits with sizes ranging from #40 to #3,0.8",
                         "104,hammer,12oz carpenters hammer,0.75",
                         "105,hammer,14oz carpenters hammer,0.875",
-                        "106,hammer,18oz carpenter hammer,1.0",
+                        "106,hammer,18oz carpenters hammer,1.0",
```

Comment:
Thanks @EMsnap for the contribution, but i'm confused why we need to change this line ?